### PR TITLE
Adding a Snowflakes-Control

### PR DESCRIPTION
**Pull request by timunie**
## What does the pull request do?

**Scope of this PR:**
- [ ] fix or update to an existing sample
- [x] add a new sample

## Checklist

**If this is a new Sample**
- [x] Added a ReadMe-file
- [x] Updated the landing page
- [x] Consider submitting a PR to https://github.com/AvaloniaUI/avalonia-docs with a link to your sample in the related documentation page

**In any case**
- [x] Spell-checking done
- [x] Checked if all hyperlinks work
- [x] Checked if all images are visible

### CODE REVIEW

**Comment by timunie** [comment]
See it in action:

final_run

File: src/Avalonia.Samples/CustomControls/SnowflakesControlSample/README.adoc [hunk 1]
```diff
@@ -0,0 +1,677 @@
...
+Hit [Run] in your IDE and enjoy your personal winter game.
+
+image::_docs/final_run.png[]
+
+
+
+== Related 
+
+Today you learned how to create a control that renders itself. If you want to learn more about it, we suggest to take a look at other samples:
+
```
Comment (nsrahmad) [inline src/Avalonia.Samples/CustomControls/SnowflakesControlSample/README.adoc:667]: I would consider this entire paragraph redundant. Just `Related` heading and links are enough.

**Review by nsrahmad — COMMENTED** [review]
Nice demo. Some grammar and spelling suggestions.

File: src/Avalonia.Samples/CustomControls/SnowflakesControlSample/README.adoc (hunk 1 above)
Comment (timunie) [reply src/Avalonia.Samples/CustomControls/SnowflakesControlSample/README.adoc]: good catch

**Comment by timunie** [comment]
Hi @nsrahmad 

thank you for your review. I corrected some spelling, but there may be some left. if you want to help me correct them, just file a new PR and I'll double-check it then.

All the best

@BnnQ: Thank you for the nice idea 🙏

**Pull request merged**